drones/image_processing/normalization.py

```diff
@@ -0,0 +1,250 @@
+import numpy as np
+import cv2 as cv
+
+
+def min_max_norm(image: np.ndarray) -> np.ndarray:
+    """Apply min max norm on given image. It should be used with OpenCv
```

Comment:
What does it mean "It should be used with OpenCv"?

Reply:
in openCV it is better to use wider range then with ai, it is hint for future

Reply:
But what is the context of "OpenCV" here? What do you define by it or "AI"?

Reply:
this two functions apply same norm,  but in case of using neural network output should have different range. In our project, we want to make ball tracking in openCV and after it do it on neural network. This suggestion is made due to future use of this files. But I will remove this suggestion from docstring and put it on wiki instead